native/src/main/resources/META-INF/services/io.ballerina.cli.BLauncherCmd

```diff
@@ -0,0 +1 @@
+io.ballerina.consolidate.ConsolidatePackagesCommand
```

Comment:
```suggestion
io.ballerina.consolidate.ConsolidatePackagesCommand

```

Reply:
Fixed in Fixed in https://github.com/ballerina-platform/consolidate-packages-tool/commit/bd473123358220cca26188349581184775fe47ca